index.html

```diff
@@ -0,0 +1,363 @@
+            <a href="https://morganheuer.github.io/magic8ball/" target="_blank">
+              <img src="/assets/magic8ball-1.png" alt="8ball">
+            </a>
+            <h4>Magic 8-Ball</h4>
+            <p>I brought this retro toy to the web, as featured in Skillcrush's JavaScript course. With the click of a button, users can seek advice from the Magic 8-Ball, and receive a prediction from the mysterious oracle inside!</p>
+            <br>
+            <p><strong>Built with: </strong>HTML, CSS, JavaScript, JQuery</p>
+            <br>
+            <a href="https://github.com/morganheuer/magic8ball" target="_blank"><p>Check out the code files here!</p></a>
+          </div> -->
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <div class="content-wrapper">
+      <h2 class="title">Let's Connect</h2>
+      <div id="contact">
+        <div class="connect">
+          <p>
+            <strong>email me</strong>
+          </p>
+          <br>
+          <div class="social-icons">
+            <p>
+              <a href="mailto:morganheuer@gmail.com" target="_blank">
+                <i class="fas fa-envelope-square">
+                </i>
+              </a>
+            </p>
+          </div>
+        </div>
+        <div class="connect">
+          <p>
+            <strong>follow me</strong>
+          </p>
+          <br>
+          <div class="social-icons">
+            <p>
+              <a href="https://github.com/morganheuer" target="_blank">
+                <i class="fab fa-github-square">
+                </i>
+              </a>
+            </p>
+            <p>
+              <a href="https://www.linkedin.com/in/morganheuer/" target="_blank">
+                <i class="fab fa-linkedin">
+                </i>
+              </a>
+            </p>
+          </div>
+        </div>
+        <div class="connect">
+          <p>
+            <strong>hire me</strong>
+          </p>
+          <br>
+          <div class="social-icons">
+            <p>
+              <a href="https://drive.google.com/file/d/1xvGalW_OBg34UCv0nRsXuTpWbBqJQjQJ/view?usp=sharing" target="_blank">
+                <i class="fas fa-briefcase">
+                </i>
+              </a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="design">
+      <p>Designed &amp; Developed with &hearts; by <br>Morgan Della // 2020</p>
+    </div>
+  </footer>
+  <script src="js/vendor/modernizr-3.11.2.min.js"></script>
+  <script src="js/plugins.js"></script>
+  <script src="js/main.js"></script>
+
+  <!-- Google Analytics: change UA-XXXXX-Y to be your site's ID. -->
+  <script>
+    window.ga = function () { ga.q.push(arguments) }; ga.q = []; ga.l = +new Date;
+    ga('create', 'UA-XXXXX-Y', 'auto'); ga('set', 'anonymizeIp', true); ga('set', 'transport', 'beacon'); ga('send', 'pageview')
+  </script>
+  <script src="https://www.google-analytics.com/analytics.js" async></script>
```

Comment:
Be careful when copy-pasting things over. It looks like this Google Analytics block is not quite set up. Probably no harm here, but copying over JavaScript that you're not sure what it's doing could lead you to accidentally introduce malicious code onto your site!

A small note - the README did ask for you not to use any external CSS frameworks or JavaScript. It's not a huge deal; it's clear you still exercised the skills we're looking for. But it's good practice to check with your boss/instructor/client/whoever when you're writing something that goes against their specifications. You did a great job checking with me that it was OK to have it as one big page, make sure to check for other things like this in the future too!